### ISSUE TITLE
Straight leg raising produces pain

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9847]
**Preferred term label:**
Straight leg raising produces pain

**Synonyms**

**Definition (free text, please give PubMed ID)**
Straight refer to the knee being extended 

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Spinal problems such as spinal epidural abscess

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000066
- label: Triggered by extension of leg
- parents: [Triggered by (HP:0025204)](https://hpo.jax.org/app/browse/term/HP:0025204)
- pmid: 
- def: This term refers to pain that is triggered by extending (straightening) the leg.
- comment: This feature may suggest spinal problems such as spinal epidural abscess.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9847
- synonyms: Straight leg raising produces pain(NONE)

**Comment by MickeySegal** [comment]
The term "straight leg raising" is often used clinically and would be helpful to be included as a synonym.
